.github/workflows/ci.yml

```diff
@@ -133,6 +133,8 @@ jobs:
       inputs.e2e_polybft_test || 
       (github.event_name == 'pull_request' && github.ref != 'refs/heads/feature/apex-bridge') || 
       (github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/develop' || github.ref != 'refs/heads/feature/apex-bridge'))
+    secrets:
+      PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
Reminder: `CI` a.k.a `Blade CI`, will not be triggered on the `feature/apex-bridge` branch. For now, this needs to be in `ApexBridge CI`.